src/sidebar/cross-origin-rpc.js

```diff
@@ -93,8 +107,34 @@ function start(store, settings, $window) {
   }
 }
 
+/**
+ * Queues all incoming RPC requests so they can be handled later.
+ *
+ * @param {object} event - RPC event payload
```

Comment:
```suggestion
 * @param {MessageEvent} event - RPC event payload
```

Since we expect this function to be called with the `event` from a "message" event on a window, you can be more specific about the type. See also the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent).